vhd/packer/configure-windows-vhd.ps1

```diff
@@ -203,8 +204,7 @@ function Install-WindowsPatches {
             # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
             # Find a specific patch at https://www.catalog.update.microsoft.com/Search.aspx?q=kb5005625
-            # KBKB5015880 Contains 2022 7C patches for Windows server 2019
-            $patchUrls = @("https://catalog.s.download.windowsupdate.com/c/msdownload/update/software/updt/2022/07/windows10.0-kb5015880-x64_1b8eadaa8f12dacd77342353c257e236e167a802.msu")
+            $patchUrls = @()

```

Comment:
Is this deletion related to the new signedscripts package, or just general cleanup?

Reply:
No, this deletion is related to https://github.com/Azure/aks-engine/pull/4978/files#diff-70e0f93cdd528a80723f7809a66d9f64a323e66e162393d144007318a7c09a46R5

We were previously installing on of the `7C` patch which is the optional updates from July. 
I updated the base image we use for the packer build to include the newest security patches (which are newer than these) so we don't need to install the optional updates anymore.